pkg/test/resourcefixture/testdata/basic/batch/v1alpha1/batchjob/batchjob-network/create.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: batch.cnrm.cloud.google.com/v1alpha1
+kind: BatchJob
+metadata:
+  name: batchjob-network-${uniqueId}
+spec:
+  projectRef:
+    external: ${projectId}
+  location: us-central1
+  priority: 10
+  allocationPolicy:
+    location:
+      allowedLocations:
+        - "zones/us-central1-a"
```

Comment:
Is the zones prefix required?  I guess we can always relax it over time anyway, so not a blocker...